services/web/server/src/simcore_service_webserver/projects/projects_api.py

```diff
@@ -199,14 +200,20 @@ async def _start_dynamic_service(
 ):
     if not _is_node_dynamic(service_key):
         return
-    project_running_nodes = await director_v2_api.list_dynamic_services(
+    project_running_nodes: list[DataType] = await director_v2_api.list_dynamic_services(
         request.app, user_id, f"{project_uuid}"
     )
 
+    project_active_running_nodes = [
+        node
+        for node in project_running_nodes
+        if node["service_state"] not in ["failed", "completed"]
```

Comment:
so we also have completed ones??? that is very interesting...
@GitHK what is the concept here. I think this calls return the director-v2 tracked services correct? why do we see completed? I had the feeling they were removed from the list at some point, is that not true?
I'm ok for now with the fix, but I am a bit afraid that we can get failed services with a still running sidecar that are eating resources.

Reply:
The name of this function is so outdated, with respect to what it returns. It's a list of states for the dynamic services (legacy and modern). Nothing more.

Had a chat with Matus and I think in this case he should only consider services which are started/starting when computing the count.